btk/measure.py

```diff
@@ -74,27 +99,33 @@ def basic_measure(batch, idx, channels_last=False, **kwargs):
     Returns:
             dict containing catalog with entries corresponding to measured peaks.
     """
-    if isinstance(batch["blend_images"], dict):
-        raise NotImplementedError("This function does not support the multi-resolution feature.")
-
     channel_indx = 0 if not channels_last else -1
-    coadd = np.mean(batch["blend_images"][idx], axis=channel_indx)
+    # We differentiate between the normal case and the multiresolution case
+    if isinstance(batch["blend_images"], dict):
+        surveys = kwargs.get("surveys", None)
+        survey_name = surveys[0].name
+        coadd = np.mean(batch["blend_images"][survey_name][idx], axis=channel_indx)
+        wcs = batch["wcs"][survey_name]
+    else:
+        coadd = np.mean(batch["blend_images"][idx], axis=channel_indx)
+        wcs = batch["wcs"]
 
     # set detection threshold to 5 times std of image
     threshold = 5 * np.std(coadd)
     coordinates = peak_local_max(coadd, min_distance=2, threshold_abs=threshold)
 
     # construct catalog from measurement.
     catalog = astropy.table.Table()
-    catalog["x_peak"] = coordinates[:, 1]
-    catalog["y_peak"] = coordinates[:, 0]
+    catalog["ra"], catalog["dec"] = wcs.pixel_to_world_values(coordinates[:, 1], coordinates[:, 0])
```

Comment:
I see that you now assume that 'ra', 'dec' are always returned (even if MR feature is not used). That makes sense, since these requires an additional step from the user maybe we should mention this somewhere in the documentation (how to access `wcs` and use it to convert your pixel coordinates)?

Reply:
Yes, this will be done in the following PR.